packages/create-universal-app/bin/generate.js

```diff
@@ -51,6 +54,10 @@ const runAsyncProcess = async (spawnFunction) => {
     const packagesDir = path.join(projectDir, "packages");
     const createUniversalAppDir = path.join(packagesDir, "create-universal-app");
     await fs.remove(createUniversalAppDir);
+    if (!userAnswers.authUI) {
+      const authDir = path.join(packagesDir, "frontend/core/src/features/auth");
```

Comment:
So currently we just delete the auth feature - is there screens setup too? e.g `apps/expo/screens/auth/login` or somethign similar?

Reply:
I did have that in the previous branch, but have deleted it here because I am unsure whether we want to start specifying specific files in the apps to be deleted? And then what about the links to those pages from the home screen? I think we might need to look into this some more... might be a good opportunity for some code generation scripts